jde/modules/ROOT/pages/demo_poll_mbf_events.adoc

```diff
@@ -0,0 +1,332 @@
+= *PortX JDE Connector for Mule 4 – Demo – Poll Transaction (MBF) Events*
```

Comment:
In general, it's best to avoid abbreviations in headings - and exception would be something that's as familiar to this audience as, say, HTML is to a broader audience. Subsequently, spelling it out on first occurrence and citing the abbreviation you're going to use thereafter (as you've done) is then way to go.